build/cmake_modules/common.cmake

```diff
@@ -57,10 +57,16 @@ if(NOT WIN32)
         set(CMAKE_C_COMPILER gcc)
         set(CMAKE_CXX_COMPILER g++)
     endif()
+endif()
 
 if(NOT APPLE)
     set(GPA_COMMON_LINK_ARCHIVE_FLAG -Wl,--whole-archive)
     set(GPA_COMMON_LINK_NO_ARCHIVE_FLAG -Wl,--no-whole-archive)
 endif()
-    add_compile_options(-Wno-unknown-pragmas -Wno-strict-aliasing -Wno-non-virtual-dtor -Wno-unused-value -msse -fvisibility=hidden)
+
+add_compile_options(-Wno-unknown-pragmas -Wno-strict-aliasing -Wno-non-virtual-dtor -Wno-unused-value -fvisibility=hidden)
```

Comment:
I just noticed I missed an indentation here. Maybe you can fix it with your next change?

Reply:
yeah, we'll take care of it. Thanks.